release: Generate changelog, set version to 1.3.0 (#474)
elastic/ecctl: docs/release_notes/v1.3.0.adoc

https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_64-bit.rpm[ecctl_1.3.0_linux_64-bit.rpm]
https://download.elastic.co/downloads/ecctl/1.3.0/ecctl_1.3.0_linux_amd64.tar.gz[ecctl_1.3.0_linux_amd64.tar.gz]

[float]
[id="{p}-release-notes-v1.3.0-whats-new"]
==== What's new

* *New top level command: `ecctl comment`.* Includes commands to create, delete, list, show and update comments for any of the Elastic Cloud resources: `elasticsearch`, `kibana`, `apm`, `appsearch`, `enterprise_search`, `allocator`, `constructor`, `runner`, `proxy`.

* *New deployment command: `ecctl deployment traffic-filter`.* Includes commands to create, delete, list and show traffic filter rules and the `ecctl deployment traffic-filter association` sub-command includes two more sub-commands to create and delete associations between traffic rules and deployments.

* *New deployment command: `ecctl deployment extension`.* Includes commands to create, delete list and show custom deployment extensions such as Elasticsearch bundles or plugins.

* *Added support for data tiers in all `ecctl deployment` commands.*

[float]
[id="{p}-release-notes-v1.3.0-bug-fixes"]
==== Bug fixes

> * *Improved deployment tracking.* The deployment tracking has been improved to only fail when the plan has actually finished with a failure, instead of returning errors from in other steps of the plan.

Reviewer: Not sure if this is any better:

```suggestion
* *Improved deployment tracking.* The deployment tracking has been improved. It now fails only when the plan completes with a failure, instead of returning errors from other steps of the plan.
```

Author: Much better! Thanks!
